CMakeLists.txt

```diff
@@ -9,12 +9,16 @@ find_package(catkin QUIET)
 if(catkin_FOUND)
   message(STATUS "Found catkin")
 
-  find_package(catkin REQUIRED 
+  find_package(catkin REQUIRED
     cv_bridge
   )
 
+  find_package(catkin REQUIRED COMPONENTS
+    roscpp
+  )
+
```

Comment:
@scosar I think here is your opencv problem. You have another `find_package(catkin` after the initial one. There most only be one. I think the correct one would be:

```
find_package(catkin REQUIRED COMPONENTS
  roscpp
  cv_bridge
)
```